lib/pbench/test/unit/server/query_apis/test_dataset_search.py

```diff
@@ -136,7 +136,6 @@ def test_query(
             index,
             expected_status,
             headers=build_auth_header["header"],
-            status=expected_status,
             json=response_payload,
         )
         assert response.status_code == expected_status
```

Comment:
> This was removed to support my simplification of the mock logic

Understood.

I was just pointing out that, as long as you were making a modification to this blob of code, it would be good to also remove the redundant assertion.